python/spec/dependabot/python/file_parser/pyproject_files_parser_spec.rb

```diff
@@ -251,6 +251,14 @@
         expect(dependency_names).to include("sphinx")
       end
     end
+
+    context "with package specify source" do
+      let(:pyproject_fixture_name) { "package_specify_source.toml" }
+      subject(:dependency) { dependencies.find { |f| f.name == "black" } }
+      it "package source is specified" do
```

Comment:
I'd normally add a blank line before this one, and try to make the description read well in English, for example, "`it "specifies a package source" do`.